### ISSUE TITLE
Cannot login to ombi, no login boxes, spinning wheel instead

### ISSUE THREAD

**Issue by jhaury** [Ombi-app/Ombi#4547]
**Describe the bug**
I cannot login to Ombi, the login boxes never appear. After the Ombi page loads there is a spinning wheel gif in place of the login boxes.

**To Reproduce**
1. load the ombi webpage

**Expected behavior**
I expect the login boxes to appear so i can login.

**Screenshots**
https://imgur.com/a/TWunXQQ (in case the upload doesnt work)

![Screen Shot 2022-03-16 at 1 44 55 AM](https://user-images.githubusercontent.com/15850496/158552800-c1f9db7b-c825-45ed-bbc3-129b2fc8c662.png)

**Logs (Logs directory where Ombi is located)**
I dont know how to get a log from docker compose app if i cant login to it.

**Desktop (please complete the following information):**
I am on MacOS, ombi is running on a headless ubuntu server via docker compose.

**Ombi Version (please complete the following information):**
 - Version - most current as far as i am aware
- Media Server - plex, is the most current
- Database Type: SQLite i assume, its docker.

**Additional context**
none

**Comment by tidusjar** [comment]
Is this the landing page? That's checking your media server status,

Are there any errors in your browser console? F12.

If so can you press the network tab and then refresh the page and you should see something like
![image](https://cloud.githubusercontent.com/assets/6642220/23088047/eac44abc-f570-11e6-9f23-d2eb022ed059.png)


If you click the red text and then the preview tab like the image above and post the result?

**Comment by jhaury** [comment]
Yes that image was of the landing page.

I did as you asked (I hope) and here is the result. "touch_track" was the only thing that was red.

![Screen Shot 2022-03-17 at 1 37 36 PM](https://user-images.githubusercontent.com/15850496/158891184-e85c077d-9ede-4e28-926d-7a86e2084cdf.png)

**Comment by twanariens** [comment]
Seems like your connection to your media server is lost in Ombi.

If you go to Ombi, remove the /landingpage from the url and change it to /login/true
That should bypass the landing page and allows you to log in to check your Media Server configuration in Ombi

**Comment by jhaury** [comment]
Thank you very much for that bit of info. That enabled me to login so I could update my settings, turns out there was an IP change recently. Solved!

Thanks!

**Comment by shanelynn321** [comment]
Can we just stop and appreciate this post for a moment. I have scraped and restored backups the past several hours, as I could not get Ombi to get past the landing page. I finally happened to stumble across this issue that gave the clue I didn't know I needed. I had tried /login and got sent back to /landingpage. /login/true however was the trick. My genius self forgot to update the Plex IP address when I upgraded the Plex hardware about a month ago.

A timeout should definitely be implemented so that users aren't stuck staring at a circle still there a half hour later when the user returns.
